docs/core/deploying/deploy-with-vs.md

```diff
@@ -18,7 +18,7 @@ The following sections show how to use Microsoft Visual Studio to create the fol
 - Self-contained deployment
 - Self-contained deployment with third-party dependencies
 
-For information on using Visual Studio to develop .NET Core applications, see [Prerequisites for .NET Core on Windows](../windows-prerequisites.md#prerequisites-to-develop-net-core-apps-with-visual-studio).
+For information on using Visual Studio to develop .NET Core applications, see [Prerequisites for .NET Core on Windows](../setup/index.md).
```

Comment:
This should go here no?
https://review.docs.microsoft.com/en-us/dotnet/core/setup/install-sdk?branch=pr-en-us-15569&pivots=os-windows#install-with-an-installer

So title and link should be changed